From gregkh@suse.de  Sun Feb  7 20:43:28 2010
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Sun,  7 Feb 2010 08:51:51 -0800
Subject: Staging: comedi: ssc_dnp: fixed a brace coding style issue
To: gregkh@suse.de
Message-ID: <1265561511-25543-1-git-send-email-gregkh@suse.de>


Fixed a coding style issue.

Created during the keynote presentation at FOSDEM 2010

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/ssv_dnp.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/comedi/drivers/ssv_dnp.c
+++ b/drivers/staging/comedi/drivers/ssv_dnp.c
@@ -300,11 +300,11 @@ static int dnp_dio_insn_config(struct co
 
 	/* read 'old' direction of the port and set bits (out=1, in=0)             */
 	register_buffer = inb(CSCDR);
-	if (data[0] == COMEDI_OUTPUT) {
+	if (data[0] == COMEDI_OUTPUT)
 		register_buffer |= (1 << chan);
-	} else {
+	else
 		register_buffer &= ~(1 << chan);
-	}
+
 	outb(register_buffer, CSCDR);
 
 	return 1;
